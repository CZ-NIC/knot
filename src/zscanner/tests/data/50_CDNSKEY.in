$ORIGIN	.
$TTL	1

; OK
@	CDNSKEY	0	0	0	AA==		; The simplest variant
@	CDNSKEY	65535	255	255	AA==		; Maximal numbers
@	CDNSKEY	0	0	RSAMD5	AA==		; Algorithm mnemonic
@	CDNSKEY	0	0	0	Zm8=		; One char padding
@	CDNSKEY	0	0	0	Zm9v		; Without padding
@	CDNSKEY	0	0	0	Zm9vYg==	; Two base64 blocks
@	CDNSKEY	0	0	0	Zm9v YmE=	; Two base64 blocks with blank space between them
@	TYPE60	\# 5 0000000000				; TYPE + Hexadecimal rdata
@	TYPE60	0	0	0	AA==		; TYPE
@	cdnskey	0	0	0	AA==		; Type in lower-case

; KO
@	CDNSKEY
@	CDNSKEY						; Empty rdata
@	CDNSKEY	\# 0					; Hex empty rdata
@	CDNSKEY	65536	0	0	AA==		; Type overflow
@	CDNSKEY	0	256	0	AA==		; Key tag overflow
@	CDNSKEY	0	0	256	AA==		; Algorithm overflow
@	CDNSKEY	0	0	0	A		; Continuous block length must be multiple of 4
@	CDNSKEY	0	0	0	AB		; Continuous block length must be multiple of 4
@	CDNSKEY	0	0	0	ABC		; Continuous block length must be multiple of 4
@	CDNSKEY	0	0	0	AA ==		; Continuous block length must be multiple of 4
@	CDNSKEY	0	0	0	A===		; Bad padding
@	CDNSKEY	0	0	0	=		; Bad padding
@	CDNSKEY	0	0	0	==		; Bad padding
@	CDNSKEY	0	0	0	===		; Bad padding
@	CDNSKEY	0	0	0	====		; Bad padding
@	CDNSKEY	0	0	0			; Missing item
