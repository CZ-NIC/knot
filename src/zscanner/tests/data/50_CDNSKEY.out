OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003C
RDATA=0000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003C
RDATA=FFFFFFFF00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003C
RDATA=0000000100
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003C
RDATA=00000000666F
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003C
RDATA=00000000666F6F
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003C
RDATA=00000000666F6F62
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003C
RDATA=00000000666F6F6261
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003C
RDATA=0000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003C
RDATA=0000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=003C
RDATA=0000000000
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_NUMBER16_OVERFLOW
------
WARNG=ZS_NUMBER8_OVERFLOW
------
WARNG=ZS_NUMBER8_OVERFLOW
------
WARNG=ZS_BAD_BASE64_CHAR
------
WARNG=ZS_BAD_BASE64_CHAR
------
WARNG=ZS_BAD_BASE64_CHAR
------
WARNG=ZS_BAD_BASE64_CHAR
------
WARNG=ZS_BAD_BASE64_CHAR
------
WARNG=ZS_BAD_RDATA
------
WARNG=ZS_BAD_RDATA
------
WARNG=ZS_BAD_RDATA
------
WARNG=ZS_BAD_RDATA
------
WARNG=ZS_BAD_RDATA
------
